src/test.ts

```diff
@@ -10,157 +10,136 @@ const tscPath = new URL(
   import.meta.url,
 );
 
-function normalizeLineEndings(text: string): string {
-  return text.replace(/\r\n?/g, "\n");
+function normalizeLineEndings(text: string) {
+  text.replace(/\r\n?/g, "\n");
 }
 
-function compareToBaselines(baselineFolder: URL, outputFolder: URL) {
-  let baselineFiles: string[] = [];
+function getFiles(folder: URL) {
   try {
-    baselineFiles = fs.readdirSync(baselineFolder);
+    return fs.readdirSync(folder);
   } catch {
-    // do nothing
+    return [];
   }
+}
 
-  let outputFiles: string[] = [];
+function readFileContent(filePath: URL) {
   try {
-    outputFiles = fs.readdirSync(outputFolder);
+    return normalizeLineEndings(fs.readFileSync(filePath, "utf-8"));
```

Comment:
We already stat the file before calling this and thus doesn't expect any error here. Thus we should skip try-catch here to see actual failures if happens.